terraform/aws/main.tf

```
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "~> 1.9"
+  required_version = "~> 1.5"
```

```suggestion
  required_version = "~> 1.5.7"
```

This would change the lower bound from 1.x to 1.5.x